4.5 release page (#1138)
godotengine/godot-website: collections/_release_4_5/entry-general-editor-mute-game-toggle.md

---
type: entry
section: general
subsection: editor
rank: 0
importance: 4
anchor: mute-game-toggle
title: Mute game toggle

> blockquote: TODO

Reviewer: ```suggestion
blockquote: Test in peace
```
...Maaaybe this is something? 😛 Maybe a little bit morbid, though, I dunno

(Also hey, this is one of my contributions! 🥳 )

Reviewer 2: Haha, I love "Test in peace"!

Author: I love it too! But the importance of the feature is 4, which usually don't have blockquotes. That why TODO didn't show. (i usually delete it though.)

I'll use "Test in peace" (it's great!), but I cannot assure you that it will make it to the release page though!

Reviewer: > Haha, I love "Test in peace"!

Glad you like it 😄 
It certainly fits the use case *I* had in mind when trying to add it – I often like listening to relaxing/focus music while I work, so being able to mute the game audio while working on non-audio-related parts was nice. 
That being said, I could understand people seeing a riff on "rest in peace" and assuming it has to do with some old feature being retired, or something similar.

text: |

>   Debugging can be tedious, especially if it exposes the developper to the same music over and over. One could turn off the speakers entirely, but it's not really practical, isn't it?

Reviewer 2: ```suggestion
  Debugging can be tedious, especially if it exposes the developer to the same music over and over. One could turn off the speakers entirely, but that's not really practical, is it?
```

>   Debugging can be tedious, especially if it exposes the developper to the same music over and over. One could turn off the speakers entirely, but it's not really practical, isn't it?

Reviewer 3: ```suggestion
  Debugging can be tedious, especially if it exposes the developer to the same music over and over. One could turn off the speakers entirely, but that’s not really practical, is it?
```

>   To keep everyone's sanity, we introduced the new mute game toggle, located in the Game view.

Reviewer 3: ```suggestion
  To keep everyone’s sanity, we introduced the new mute game toggle, located in the Game view.
```
